charts/db-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
-kubeVersion: ">= 1.19-prerelease <= 1.22-prerelease"
+kubeVersion: ">= 1.19-prerelease <= 1.23-prerelease"
```

Comment:
pipeline should also test a 1.23 version.

https://github.com/kloeckner-i/db-operator/blob/master/.github/workflows/build-and-test.yaml#L85

Reply:
done, added `v1.23.1`